examples/with-external-scoped-css/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Could you make this file in line with https://github.com/zeit/next.js/blob/master/examples/basic-css/package.json

`name` same as example directory
`license` ISC
`author` empty

Also `next`, `react` and `react-dom` should be inside the `dependencies`. Not `devDependencies`.

`"next": "^2.0.0-beta.17"` can be changed to `"next": "^2.0.0-beta"`